source/developers-guide/rest-api/extend-api-resource/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Extend a REST API resource
+github_link: developers-guide/rest-api/extend-api-resource/index.md
+shopware_version: 5.2.17
+indexed: true
+tags:
+  - extend
+  - api
+  - Plugin
+  - resource
+group: Developer Guides
+subgroup: REST API
+menu_title: Extend a REST API resource
+menu_order: 140
+---
+
+<div class="toc-list"></div>
+
+## Introduction
+This article will provide a small example on how to extend the existing REST API resources. As of shopware 5.2.17 the resources are loaded as services to the dependency injection container and can be easily decorated.
+
+## Plugin files
+For this example plugin we only need a few files. For more information about necessary files and the 5.2 plugin system
```

Comment:
You only need a few files for this example plugin. ... and the Shopware 5.2 plugin system see ...